priv/repo/migrations/20170419210307_create_user.exs

```diff
@@ -0,0 +1,15 @@
+defmodule RemoteRetro.Repo.Migrations.CreateUser do
+  use Ecto.Migration
+
+  def change do
+    create table(:users) do
+      add :google_user_info, :jsonb
```

Comment:
I didn't think about that, but that makes sense and I agree with you.  Will make some changes